drivers/focuser/lacerta_mfoc.cpp

```diff
@@ -61,11 +61,11 @@ void lacerta_mfoc::ISGetProperties(const char *dev)
 
     INDI::Focuser::ISGetProperties(dev);
 
-    defineProperty(&TempTrackDirSP);
-    loadConfig(true, TempTrackDirSP.name);
+    defineProperty(TempTrackDirSP);
+    TempTrackDirSP.load();
```

Comment:
Do not load property config in ISGetProperties(). Just remove any reference to loadProperties.